app/cdash/xml_handlers/abstract_xml_handler.php

```diff
@@ -93,7 +93,7 @@ public static function validate(string $path): array
             $validation_errors = libxml_get_errors();
             foreach ($validation_errors as $error) {
                 if ($error->level === LIBXML_ERR_ERROR || $error->level === LIBXML_ERR_FATAL) {
-                    $errors[] = "ERROR: {$error->message} in {$error->file}, line: {$error->line}, column: {$error->column}";
+                    $errors[] = "ISSUE: {$error->message} in {$error->file}, line: {$error->line}, column: {$error->column}";
```

Comment:
Warning?

Reply:
That would make sense and it could nicely update to a full error if we needed it to.